index.html

```diff
@@ -38,7 +38,124 @@ <h2>LET'S CONNECT</h2>
+    <h2>Tonic</h2>
+    <ul class="profesional">
+        <li class="item-can">CANOPY </li>
+        <li class="item-tag">&#9679; Back End Dev &#9679;</li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+
+<div class="card">
+    <img src="img/Snapshoot Portfolio(3).png" alt="">
+    <h2>Multi-Post
+        Stories</h2>
+    <ul class="profesional">
+        <li class="item-can">CANOPY</li>
+        <li class="item-tag">&#9679; Back End Dev &#9679; </li>
+        <li class="item-tag">2015</li>
+    </ul>
+    <p>A daily selection of 
+        privately personalized reads; no accounts
+         or sign-ups required.</p>
+        <ul class="category">
+            <li class="tag">html</li>
+            <li class="tag">css</li>
+            <li class="tag">javascript</li>
+        </ul>
+    <button>see project</button>
+</div>
+
+<div class="about-me">
+    <h1 class="about-myself">About<br>
+        Myself</h1>
+    <p>Hello I’m a software developer! I can help you build a product ,
+         feature or website Look through some of my work and experience!
+         If you like what you see and have a project you need coded,
+          don’t hestiate to contact me..</p>
+          <h3 class="connect-header">LET’S CONNECT</h3>
+          <div class="social-media">
+            <ul>
+                <li><a href="#"><i class=" fa fa-twitter" ></i></a></li>
+                <li><a href="#"><i class="fa fa-linkedin"  ></i></a></li>
+                <li><a href="#"><i class="fa brand fa fa-medium" ></i></a></li>
+                <li><a href="#"><i class="fa fa-github" aria-hidden="true"></i></a></li>
+                <li><a href="#"><i class="fa brand fa fa-angellist" ></i></a></li>
+            </ul>   
+         </div> 
+          <button>Get my resume</button>
+          <div class="language-header">
+          <h3 class="language-heading">Languages<i class="fa fa-angle-down"></i></i></h3>
+        </div>
+        <ul class="list-language">
+            <li class="tag-lang"><img src="/img/skill1.png" alt="html">JavaSript</li>
+            <li class="tag-lang"><img src="/img/skill2.png" alt="html">HTML</li>
+            <li class="tag-lang"><img src="/img/skill3.png" alt="html">CSS</li>
+        </ul> 
+        <ul class="frame-work">
+            <li class="skill">
+                <h1 class="skill-title">Frameworks<i class="fa fa-angle-right" ></i></h1>
+                <hr class="line">  
+            </li>
+            <li class="skills">
+                <h1 class="second-title">Skills<i  class="fa fa-angle-right" ></i></h1> 
+                <hr class="line">
+            </li>
+        </ul>
+    </section>
```

Comment:
- [ ] Kindly as suggested by the previous reviewer, the ```work-section```  should be contained in one ```section``` tag and the ```about-section``` should be contained in another single ```section``` tag 😎. 

So, kindly use one section for the work section and another for the about section as they are two different **sections** they shouldn't be in the same ```section``` tag. 😉😉

_You may follow this [link](https://www.geeksforgeeks.org/html-section-tag/) for more on the ```section``` tag_